README.md

```diff
@@ -52,6 +52,35 @@ This project comes with a command that generate the JavaScript version of all yo
 
     php artisan lang:js -c
 
+**(optional)Use [gulp](http://gulpjs.com/) to publish:**
+
+install `gulp-shell` from https://github.com/sun-zheng-an/gulp-shell with `npm install --save-dev gulp-shell` 
+and then run it directly in your `gulpfile.js`:
+
+    var shell = require('gulp-shell');
+
+    //......
+
+    gulp.task('langJs', shell.task('php artisan lang:js -c public/js/messages.js'));
```

Comment:
@okaufmann please, instead of using 4 spaces for code use ````` to wrap the code and add the _language hint_ to it. This will tell GH to add the proper syntax highlighting colours. Example:

**Markdown:**

    ```js
    var shell = require('gulp-shell');
    ```

**Results:**

```js
var shell = require('gulp-shell');
```